prodmgr/main.py

```diff
@@ -157,8 +215,7 @@ def run_install_utility(image_name, image_version, args, remaining_args):
     try:
         check_call(podman_command)
     except CalledProcessError as cpe:
-        raise ProdmgrError(f'Running install utility failed: {cpe}')
-
+        raise ProdmgrError(f'Running {image_name} failed: {cpe}')
```

Comment:
PEP8/pycodestyle wants an extra blank line before the next function definition.

```suggestion
        raise ProdmgrError(f'Running {image_name} failed: {cpe}')

```